pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1034,11 +1090,109 @@ func (vs *VSphere) NodeAdded(obj interface{}) {
 func (vs *VSphere) NodeDeleted(obj interface{}) {
 	node, ok := obj.(*v1.Node)
 	if node == nil || !ok {
-      glog.Warningf("NodeDeleted: unrecognized object %+v", obj);
+		glog.Warningf("NodeDeleted: unrecognized object %+v", obj);
 		return
 	}
 
 	glog.V(4).Infof("Node deleted: %+v", node)
 	vs.nodeManager.UnRegisterNode(node)
 }
 
+// convertVolPathsToDevicePaths removes cluster or folder path from volPaths and convert to canonicalPath
+func (vs *VSphere) convertVolPathsToDevicePaths(ctx context.Context, nodeVolumes map[k8stypes.NodeName][]string) (map[k8stypes.NodeName][]string, error) {
```

Comment:
can you move this to Util. vsphere.go should only contain the major functionality. All utilities should in vsphere_util.go

Reply:
Agreed. This function needs methods defined on struct VSphere, so I think to keep the function in this file